### PR TITLE
Schedulerapi binding fails when the server is restarted 

### PR DESCRIPTION
**Pull request by Aminelouati**

### CODE REVIEW

**Comment by Aminelouati** [comment]
test it again

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 2 issues

1. MAJOR [SchedulingMethodImpl.java#L219](https://github.com/ow2-proactive/scheduling/blob/1d1d7dc28defd1d7bc8fff7e4e3bfaa3f7b93d3f/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingMethodImpl.java#L219): Refactor this code to not nest more than 3 if/for/while/switch/try statements. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS134)
1. MAJOR [SchedulingMethodImpl.java#L258](https://github.com/ow2-proactive/scheduling/blob/1d1d7dc28defd1d7bc8fff7e4e3bfaa3f7b93d3f/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingMethodImpl.java#L258): This block of commented-out lines of code should be removed. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingMethodImpl.java [hunk 1]
```diff
@@ -246,8 +255,7 @@ public int schedule() {
 
                         if (currentPolicy.isTaskExecutable(nodeSet, taskDescriptor)) {
                             // load and Initialize the executable container
-                            loadAndInit(internalTask);
-
+                            // loadAndInit(internalTask);
```
Comment (fviale) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingMethodImpl.java]: please remove this commented code

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 2 issues

1. MAJOR [SchedulingMethodImpl.java#L219](https://github.com/ow2-proactive/scheduling/blob/10c6738272cd818e183fdf632f551f513e083e7e/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingMethodImpl.java#L219): Refactor this code to not nest more than 3 if/for/while/switch/try statements. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS134)
1. MAJOR [SchedulingMethodImpl.java#L258](https://github.com/ow2-proactive/scheduling/blob/10c6738272cd818e183fdf632f551f513e083e7e/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingMethodImpl.java#L258): This block of commented-out lines of code should be removed. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 1 issue

1. MAJOR [SchedulingMethodImpl.java#L219](https://github.com/ow2-proactive/scheduling/blob/ce0fe01d96ce07924e5777617fea95d1fddb2289/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingMethodImpl.java#L219): Refactor this code to not nest more than 3 if/for/while/switch/try statements. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS134)

**Pull request merged**